projects/packer_ami_builder/resources/production/templates/ami_building_policy.tpl

```diff
@@ -1,39 +1,107 @@
 {
-  "Version": "2012-10-17",
-  "Statement": [{
-      "Effect": "Allow",
-      "Action" : [
-        "ec2:AttachVolume",
-        "ec2:CreateVolume",
-        "ec2:DeleteVolume",
-        "ec2:CreateKeypair",
-        "ec2:DeleteKeypair",
-        "ec2:DescribeSubnets",
-        "ec2:CreateSecurityGroup",
-        "ec2:DeleteSecurityGroup",
-        "ec2:AuthorizeSecurityGroupIngress",
-        "ec2:CreateImage",
-        "ec2:CopyImage",
-        "ec2:RunInstances",
-        "ec2:TerminateInstances",
-        "ec2:StopInstances",
-        "ec2:DescribeVolumes",
-        "ec2:DetachVolume",
-        "ec2:DescribeInstances",
-        "ec2:CreateSnapshot",
-        "ec2:DeleteSnapshot",
-        "ec2:DescribeSnapshots",
-        "ec2:DescribeImages",
-        "ec2:RegisterImage",
-        "ec2:CreateTags",
-        "ec2:ModifyImageAttribute"
-      ],
-      "Resource" : "*",
-        "Condition": {
-          "StringEquals": {
-            "ec2:vpc": "arn:aws:ec2:${region}:${account_id}:vpc/${vpc_id}"
-          }
-       }
-    }
-  ]
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "NonResourceBasedReadOnlyPermissions",
```

Comment:
Can these `Sid` be made more descriptive? For example, I think many of these actions are not read-only. It'd be good if the Sid could help us to distinguish between the groups of actions there are here.